src/js/components/HealthTab.js

```diff
@@ -131,7 +131,7 @@ class HealthTab extends React.Component {
       <div>
         <FilterHeadline
           currentLength={visibleData.length}
-          inverseStyle={false}
+          inverseStyle={true}
```

Comment:
I figured that the `FilterHeadline` is so specific that there is really no scenario we will be trying to apply other classes to it. Also the inverse styling is applied to it in a complicated, I think to expose all the styles is unnecessary (especially given that in this project we would never need to use them)